_layouts/default.html

```diff
@@ -3,15 +3,23 @@
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 		<title>cylc - {{ page.title }}</title>
-		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="{{ site.url }}/assets/css/main.css">
+		{%- if page.styles -%}
+		{% for style in page.styles %}
+		<link rel="stylesheet" type="text/css" href="{{ site.url }}/{{ style }}">
+		{%- endfor -%}
+		{% endif %}
 	</head>
 	<body>
-        {% include header.html %}
+    {% include header.html %}
 		<div id="wrapper">
-			<div class="page">
```

Comment:
@sadielbartholomew this was moved to `page.html`. Which means that if `users.md` extended `default` directly, your map area could occupy the whole page width.